website/pages/docs/internals/telemetry.mdx

```diff
@@ -105,7 +105,8 @@ These metrics represent runtime aspects of the running Vault instance.
 | `vault.runtime.malloc_count`      | Cumulative count of allocated heap objects                                                                                                                   | objects    | gauge   |
 | `vault.runtime.num_goroutines`    | Number of goroutines. This serves as a general system load indicator worth establishing a baseline and thresholds for alerting.                              | goroutines | gauge   |
 | `vault.runtime.sys_bytes`         | Number of bytes allocated to Vault. This includes what is being used by Vault's heap and what has been reclaimed but not given back to the operating system. | bytes      | gauge   |
-| `vault.runtime.total_gc_pause_ns` | The total garbage collector pause time since Vault was last started                                                                                          | ms         | summary |
+| `vault.runtime.total_gc_pause_ns` | The total garbage collector pause time since Vault was last started                                                                                          | ns         | gauge |
+| `vault.runtime.gc_pause_ns` | Total duration of the last garbage collection run | ns | summary |
```

Comment:
The only reference I can find to this metric is located [here](https://github.com/hashicorp/vault/blob/9caac4d838dca0e4a347f83a9804eb9106f8453a/vendor/github.com/armon/go-metrics/metrics.go#L267). Since it's a sample, is this description correct?

Reply:
Nope. Changed it to sample. 

Good catch